src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiEndpoint: 'https://localhost:44378/api/v1'
+  apiEndpoint: 'http://localhost:44378/api/v1',
```

Comment:
IIS has SSL enabled, what is the reason of changing it to http?

Reply:
WebApp in dev environment doesn't run with SSL enabled thus causes errors

Reply:
Fixed